modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -51,7 +51,7 @@ class NDB_Form_Imaging_Browser extends NDB_Form
         // allow only to view own site data
         $timePoint =& TimePoint::singleton($_REQUEST['sessionID']);
 
-        return ($user->hasPermission('imaging_browser_view_allsites') || ((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs'))) && $user->hasPermission('imaging_browser_view_site')));
+        return ($user->hasPermission('imaging_browser_view_allsites') || ((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs'))) && $user->hasPermission('imaging_browser_view_site')) || $user->hasPermission('imaging_browser_phantom_data'));
```

Comment:
yes @jacobpenny 
this is intended because phantom visits take place at many sites (in fact, all sites), so I am treating it like if it is the `imaging_browser_view_allsites` permission, but modifying the basic query by appending to it the `AND` statement that the candidate to be displayed has Entity_type='Scanner' which makes it a phantom data 

(so someone with this permission has access to everything site-wise, but the query would limit the results returned to only those of a phantom dataset)